make/jvm/pom.xml

```diff
@@ -29,6 +29,7 @@ SOFTWARE.
   <version>1.0-SNAPSHOT</version>
   <properties>
     <eo.version>0.29.1</eo.version>
+    <stack-size>256M</stack-size>
```

Comment:
@Graur Do we need such a big stack size is you fixed the original problem? 
Maybe we can reduce it up to `32M` ? What do you think?

Reply:
@volodya-lombrozo I forgot to get rid of it, thanks!